apps/dashboard/app/(app)/ratelimits/page.tsx

```
@@ -37,6 +37,8 @@ export default async function RatelimitOverviewPage() {
     return redirect("/new");
   }
 
+  console.log(workspace.ratelimitNamespaces,`workspace.ratelimitNamespaces`)
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)